sdk/nodejs/eks/cluster.ts

```diff
@@ -139,7 +139,7 @@ export class Cluster extends pulumi.CustomResource {
     /**
      * Nested attribute containing `certificate-authority-data` for your cluster.
      */
-    public /*out*/ readonly certificateAuthority!: pulumi.Output<outputs.eks.ClusterCertificateAuthority>;
+    public /*out*/ readonly certificateAuthorities!: pulumi.Output<outputs.eks.ClusterCertificateAuthority[]>;
```

Comment:
This is also a breaking change - and I think may even break `@pulumi/eks`.  Similar question on whether this is expected/necessary?

We may need to invest in the ability in `tfbridge` to project fields like this as both singular and plural to avoid this sort of change being a breaking change?